apps/server/src/infra/identity-management/keycloak-configuration/service/keycloak-migration.service.ts

```diff
@@ -2,7 +2,7 @@ import UserRepresentation from '@keycloak/keycloak-admin-client/lib/defs/userRep
 import { Injectable } from '@nestjs/common';
 import { LegacyLogger } from '@src/core/logger';
 import { AccountService } from '@modules/account/services/account.service';
-import { AccountDto } from '@modules/account/services/dto';
+import { Account } from '@src/modules/account/domain';
```

Comment:
This produce a dependency cycle. This module should not import any from account. Same for the service it self.
Please speak about it.